sopel/config/core_section.py

```diff
@@ -1232,24 +1232,49 @@ def homedir(self):
 
     :default: ``ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384``
 
-    The default is the `Mozilla Intermediate configuration`_ as of 2022-02.
-    This parameter is in OpenSSL format, see the `ciphers manual page`_ for
-    your version. This setting cannot be used to configure TLS 1.3.
+    The default is the `Mozilla Intermediate configuration`__ as of February
+    2022. This parameter is in OpenSSL format; see the `ciphers manual page`__
+    for your version. This setting cannot be used to configure TLS 1.3.
 
-    .. _`Mozilla Intermediate configuration`: https://wiki.mozilla.org/Security/Server_Side_TLS
-    .. _`ciphers manual page`: https://www.openssl.org/docs/manmaster/man1/ciphers.html
+    .. __: https://wiki.mozilla.org/Security/Server_Side_TLS
+    .. __: https://www.openssl.org/docs/manmaster/man1/ciphers.html
 
     .. note::
 
-        Cipher selection is also subject to the available SSL/TLS versions.
-        Sopel currently sets the minimum version to TLS 1.2.
+        Cipher selection is also subject to the available SSL/TLS versions,
```

Comment:
This is another comma splice like the one I fixed before. Should be a semicolon.